inotify/adapters.py

```diff
@@ -48,17 +50,17 @@ def __init__(self, type_name, event):
 
 
 class Inotify(object):
-    def __init__(self, paths=[], block_duration_s=_DEFAULT_EPOLL_BLOCK_DURATION_S):
-        self.__block_duration = block_duration_s
+    def __init__(self, paths=[], block_duration_ms=_DEFAULT_POLL_BLOCK_DURATION_MILLISECONDS):
+        self.__block_duration = block_duration_ms
```

Comment:
You can also get rid of the `__get_block_duration` method, its treating `self.__block_duration` like it might be a function (it won't) and it talks about it like its in seconds, which it isn't anymore

Reply:
Yeah I don't like that method. I guess I was trying to keep the API somewhat the same, but I don't think this fork will ever get merged so I'll just take it out